src/components/ContentWrapper.jsx

```diff
@@ -328,6 +327,9 @@ const ContentWrapper = (props) => {
       <ChangesNotAppliedModal experimentId={experimentId} />
 
       <Sider
+        style={{
+          overflow: 'auto', height: '100vh', position: 'fixed', left: 0,
```

Comment:
If you remove `position` and `left`, and change `height` to `minHeight`, it you can remove the CSS in `<Layout>` below and still fix the sidebar. The "Teams - Careers - Contacts" part will be put at the bottom though. This is because of `height: 100%` on line 340. To remedy that, maybe you can tweak the height around, or rearrange.

Reply:
with the proposed solution the clickable elements scroll out of view which is not the intended behavior